addons/mail/static/src/discuss/call/common/call_settings.js

```diff
@@ -1,15 +1,12 @@
-import { ActionPanel } from "@mail/discuss/core/common/action_panel";
-
 import { Component, onWillStart, useExternalListener, useState } from "@odoo/owl";
 
 import { _t } from "@web/core/l10n/translation";
 import { browser } from "@web/core/browser/browser";
 import { useService } from "@web/core/utils/hooks";
 
 export class CallSettings extends Component {
-    static components = { ActionPanel };
     static template = "discuss.CallSettings";
-    static props = ["thread", "className?"];
+    static props = ["*"];
```

Comment:
I suppose that's due to `thread` being passed for all thread actions, right?
Honestly that's good enough. Props validation is kinda overrated. We could have defined a `threadActionProps` that's exported then imported to all thread action panels but that's more LOCs and honestly not a necessity, so `*` is good.

If the component makes use of an explicit props in the future, then we could clean the code at that point ;)